src/store/app-info/app-info.reducer.ts

```diff
@@ -46,6 +40,18 @@ export const appInfoReducer = createReducer(
     ...state,
     updateAvailablePresented: hasSeen,
   })),
+  on(UpdateAuthResult, (state: AppInfoStateModel, { authResult }) => ({
```

Comment:
just for my curiosily, how is this different from the old implementation.  Did that have its own area in the state?

Reply:
pretty sure in the old implementation, it was only ever built and retained locally within the provider itself.

Seemed nonsensical to do that when the auth result is used in numerous places due to how and when authentication is invoked, so we moved it to the state